app/components/primer/tab_container_component.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use TabContainer to create tabbed content with keyboard support.
```

Comment:
```suggestion
  # Use TabContainer to create tabbed content with keyboard support. This component requires javascript.
```

Maybe a good enough note for now about the JS dep?